lib/app_profiler/stackprof_profile.rb

```diff
@@ -4,6 +4,17 @@ module AppProfiler
   class StackprofProfile < BaseProfile
     FILE_EXTENSION = ".stackprof.json"
 
+    class << self
+      def backend_name
+        Backend::StackprofBackend.name.to_s
```

Comment:
If we want stackprof to ever be an optional dependency this should be refactored the same way we did for vernier, right?

Reply:
yes -- but I think its okay for now to refer to the backend. Let me know if you feel strongly about this, I am ok either way